nautobot/core/views/mixins.py

```diff
@@ -957,6 +958,7 @@ class ObjectChangeLogViewMixin(NautobotViewSetMixin):
 
     base_template = None
 
+    @rf_action(detail=True)
```

Comment:
Might be more clear to import this as `drf_action`

Reply:
Good point, will swap out.